### ISSUE TITLE
change the (sz %d) thing from classlisting because its bad for grepping, just show the size is enough imho

### ISSUE THREAD

**Issue by radare** [radare/radare2#13190]
```
$ git diff
diff --git a/libr/core/cbin.c b/libr/core/cbin.c
index 7d1faad87..7483bc9fe 100644
--- a/libr/core/cbin.c
+++ b/libr/core/cbin.c
@@ -2943,7 +2943,7 @@ static int bin_classes(RCore *r, int mode) {
                        }
                } else {
                        int m = 0;
-                       r_cons_printf ("0x%08"PFMT64x" [0x%08"PFMT64x" - 0x%08"PFMT64x"] (sz %"PFMT64d") class %d %s",
+                       r_cons_printf ("0x%08"PFMT64x" [0x%08"PFMT64x" - 0x%08"PFMT64x"] %6"PFMT64d" class %d %s",
                                c->addr, at_min, at_max, (at_max - at_min), c->index, c->name);
                        if (c->super) {
                                r_cons_printf (" super: %s\n", c->super);
```

**Comment by radare** [comment]
https://github.com/radare/radare2/pull/13238

**Comment by radare** [comment]
Fixed in fc63cda1ae74309b19e8a672d4d1661820904136
